xmtp_id/src/associations/serialization.rs

```diff
@@ -62,6 +71,228 @@ pub enum DeserializationError {
     InvalidAccountId,
 }
 
+impl TryFrom<IdentityUpdateProto> for UnverifiedIdentityUpdate {
+    type Error = DeserializationError;
+
+    fn try_from(proto: IdentityUpdateProto) -> Result<Self, Self::Error> {
+        let client_timestamp_ns = proto.client_timestamp_ns;
```

Comment:
something I've found helpful/cleaner with these conversions traits, since we have so many of them is to use the struct desugaring syntax:

`let IdentityUpdateProto { client_timestamp_ns, inbox_id, actions } = proto;`

which accomplishes the same thing as these assignments, but just feels better

Reply:
You're right. That is nicer. Will update.